cmd/run.go

```diff
@@ -50,8 +50,18 @@ func newRunCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// runCmd represents the run command
 	var runCmd = &cobra.Command{
 		Use:   "run",
-		Short: "Run the local Appsody environment for your project",
-		Long:  `This starts a docker based continuous build environment for your project.`,
+		Short: "Run your Appsody project in a containerized development environment.",
+		Long: `Run the local Appsody environment, starting a container based continuous build environment for your project.
+		
+Run this command from the root directory of your Appsody project`,
+		Example: `  appsody run
+  Runs your project in a containerized development environment.
+
+  appsody run --interactive
+  Runs your project in a containerized development environment, and attaches the standard input stream to the container. This allows you to interact with the processes within the container.
+
+  appsody run --docker-volume my-volume
```

Comment:
I think you meant `--deps-volume`? This option won't be used very often.

It would be more valuable to show examples of `-p` and `--docker-options` as we get a lot more questions about these.

Reply:
You are right! I will change this to cover the `-p` and `--docker-options` use cases instead